### ISSUE TITLE
Bogus records in database makes plugin to skip database metrics at all

### ISSUE THREAD

**Issue by antonsoroko** [freight-hub/verdaccio-openmetrics#7]
e.g. for redis database plugin  - bogus keys (members) in "ve:pkgs" key makes plugin to skip database metrics at all
(in this case `test` is some bogus record in database)
```
WARN: Failed to collect database metrics due to NotFoundError: package test's package.json not found
    at getError (/usr/local/lib/node_modules/verdaccio/node_modules/verdaccio-redis-storage/node_modules/@verdaccio/commons-api/lib/index.js:113:45)
    at getNotFound (/usr/local/lib/node_modules/verdaccio/node_modules/verdaccio-redis-storage/node_modules/@verdaccio/commons-api/lib/index.js:147:10)
    at StoragePluginManager.readPackageAsync (/usr/local/lib/node_modules/verdaccio/node_modules/verdaccio-redis-storage/lib/PackageStorage.js:204:43)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

we had some leftovers after migration - but anyway it makes sense to ignore such errors and not stop processing, imho.

thank you!

**Comment by danopia** [comment]
Hello, I attached a PR to address this by logging the bad package's names and moving on. I suppose it could be a metric instead but knowing which package has the issue is more useful I think, so the record can be cleaned up.

Are you still on Verdaccio 4? I'm going to be updating dependencies on this plugin to Verdaccio 5, and wasn't planning on making a further Verdaccio 4 release 😄

**Comment by antonsoroko** [comment]
Hi. Thank you!
I use Verdaccio 5.

>but knowing which package has the issue is more useful I think
Agree.

**Comment by danopia** [comment]
I've released this small patch as v1.1.1 and verified with my own deployment (on Google Cloud backend), getting a log line like this:

```
WARN: Failed to fetch local package "@repro/missing-package" for database metrics: no such package available
```

I'll ship 1.2.0 next targeting Verdaccio 5.

Thanks for reporting :)

**Comment by antonsoroko** [comment]
Cool. I already see it in https://www.npmjs.com/package/verdaccio-openmetrics

Thank you very much for the quick fix!
